src/resources/games-toolkit.md

```diff
@@ -1,241 +1,382 @@
 ---
 title: Casual Games Toolkit
-description: Learn more about creating app-like games in Flutter.
+description: Free & open source multiplatform 2D game development in Flutter
 ---
 
-The Flutter Casual Games Toolkit, announced at Google I/O 2022,
-pulls together new and existing resources that enable you to
-speed up development of casual games on mobile.
+The Flutter Casual Games Toolkit pulls together new and existing resources
+so you can accelerate development of games on mobile platforms.
 
-This page outlines what resources are available and
-where you can find them.
+This page outlines where you can find these available resources.
 
-For instructions on how to use the game template, please review the [game template readme][game-template-readme].
+## Why Flutter for games?
 
-## How Flutter can help
+The Flutter framework can create performant apps for six target platforms
+from the desktop to mobile devices to the web.
 
-In the beginning, four engineers had a vision of
-developing a framework for creating performant mobile apps
-written as a single application.
-What later became Flutter has grown in many ways,
-but has always focused on writing apps.
+The benefits of cross-platform development and performance make Flutter
+a great choice for games.
 
-Games fall into roughly two categories: video games
-and app-like games. You are likely familiar with both
-types of games, though you might not have thought
-about them in quite this way.
+Video games fall into two categories: hobbyist games and casual games.
+You may be familiar with both types of games,
+though you might not have thought about them in quite this way.
 
-_Video games_ include arcade-style games: shooters,
-racing games, and so on.
-Video games are generally action-based and use a
-continual input device, like a joystick.
+_Hobbyist games_ cover games for those who dedicate time to the playing of games
+with complex interfaces, rules, and controllers.
+These players are also known as core, hardcore, or professional gamers.
+These include arcade-style games: shooters, racing games, and so on.
+These types of games could use the [Flame game engine][] built using Flutter.
```

Comment:
It seems to me, that Flame has nothing to do with being a hobbyist. It has everything to do with the speed of game play.